.github/workflows/ci.yml

```diff
@@ -174,7 +174,9 @@ jobs:
       - name: Update maven settings
         if: ${{ github.event_name != 'pull_request' || matrix.server != 'undertow-map-hot-rod' || env.GIT_HOTROD_RELEVANT_DIFF != 0 }}
         run: mkdir -p ~/.m2 ; cp .github/settings.xml ~/.m2/
-
+      - name: Prepare test providers
+        if: ${{ matrix.server == 'quarkus' || matrix.server == 'quarkus-map' }}
+        run: ./mvnw clean install -nsu -B -e -f testsuite/integration-arquillian/servers/auth-server/services/testsuite-providers -Pauth-server-quarkus
```

Comment:
I think this might not be necessary. I might be wrong but I think the testsuite providers are already built here:
https://github.com/keycloak/keycloak/blob/cdcd09ad038bd114defa564c923767755fc6d640/.github/workflows/ci.yml#L37

Reply:
Yes, they are, but we have a single artifact. To cover the fix, we need to make sure the artifact is built for Quarkus.

That is the tricky part and why we did not catch this issue before.

Reply:
Ah, right, makes sense. 👍